### PR TITLE
Promote 0.5.0 to production

### PR DESCRIPTION
**Pull request by mateoclarke**
[Release Notes](https://github.com/cityofaustin/atd-moped/releases/tag/v0.5.0)

For testing: https://moped.austinmobility.io/moped

### CODE REVIEW

**Comment by johnclary** [comment]
@mateoclarke @sergiogcx in staging, i'm unable to view/edit a project's location after creating it. I created [issue #6168](https://github.com/cityofaustin/atd-data-tech/issues/6168) for this.

I'm quite sure this was working fine as recently as Tuesday.

**Comment by sergiogcx** [comment]
> @mateoclarke @sergiogcx in staging, i'm unable to view/edit a project's location after creating it. I created [issue #6168](https://github.com/cityofaustin/atd-data-tech/issues/6168) for this.
> 
> I'm quite sure this was working fine as recently as Tuesday.

Yea I can't edit it either, checking to see if the data is persisted at all or to find out what exactly is causing the map to crash.

**Comment by sergiogcx** [comment]
So I can confirm the features are persisted in the database, the console shows this `@math.gl/web-mercator: assertion failed` error:

**Comment by johnclary** [comment]
@sergiogcx that webmercator assertion makes me think the issue would be related to the feature coordinates, but those coordinates should be fine (although the decimal places are excessive). 

I see that the `PROJECT_EXTENT_ID` of this feature is `RANDOM_FEATURE_ID`. looking at the fallback code, i would expect that the map data being logged would have an actual uuid, not the placeholder string from your fallback query. 

Screen Shot 2021-05-21 at 5 10 15 PM

**Comment by sergiogcx** [comment]
Thanks @johnclary 🙏 looking at your proposal right now

File: moped-editor/src/utils/mapHelpers.js [hunk 1]
```diff
@@ -616,11 +616,10 @@ export function useFeatureCollectionToFitBounds(
      */
     const fitViewportToBounds = viewport => {
       if (featureCollection.features.length === 0) return viewport;
-      debugger;
       const featureViewport = new WebMercatorViewport({
         viewport,
-        width: currentMap?._width ?? window.innerWidth,
-        height: currentMap?._height ?? window.innerHeight,
+        width: currentMap?._width ?? (window.innerWidth * 0.80),
```
Comment (johnclary) [inline moped-editor/src/utils/mapHelpers.js:621]: @sergiogcx this looks good on my end. confirming that project maps are rendering now. I agree there's a better way to fix this but this gets it done for now.

**Review by johnclary — APPROVED** [review]
@sergiogcx this looks good on my end. confirming that project maps are rendering now. I agree there's a better way to fix this but this gets it done for now.

File: moped-editor/src/utils/mapHelpers.js [hunk 2]
```diff
@@ -616,11 +616,11 @@ export function useFeatureCollectionToFitBounds(
      */
     const fitViewportToBounds = viewport => {
       if (featureCollection.features.length === 0) return viewport;
-
+      debugger;
```
Comment (mateoclarke) [inline moped-editor/src/utils/mapHelpers.js]: @sergiogcx, do we need to remove this `debugger` from the release canddiate?

Comment (sergiogcx) [reply moped-editor/src/utils/mapHelpers.js]: Ah, yes my bad. I forgot to push my latest commit.

Comment (sergiogcx) [reply moped-editor/src/utils/mapHelpers.js]: @mateoclarke done

**Comment by johnclary** [comment]
@amenity  heads-up that @sergiogcx's patch to fix the project summary map does degrade the way the map zooms to the project locations. not sure it will happen with every project, but the default map zoom may not quite be right (image below). 

This will get fixed with [#6183](https://github.com/cityofaustin/atd-data-tech/issues/6183). I don't think it's a show-stopper?

Screen Shot 2021-05-25 at 2 09 06 PM

**Comment by sergiogcx** [comment]
> @amenity heads-up that @sergiogcx's patch to fix the project summary map does degrade the way the map zooms to the project locations. not sure it will happen with every project, but the default map zoom may not quite be right (image below).
> 
> This will get fixed with [#6183](https://github.com/cityofaustin/atd-data-tech/issues/6183). I don't think it's a show-stopper?
> 
> Screen Shot 2021-05-25 at 2 09 06 PM

@amenity @johnclary We don't know if this map problem is going to happen in production with certainty, but this is a clear problem in staging. Created an issue to work on it for next release.

**Comment by amenity** [comment]
Agree that the zoom isn't a show-stopper, @johnclary . 

I think I would like to address the bug [we're both seeing](https://austininnovation.slack.com/archives/CNUEPKLB1/p1621968965059600) on staging: People I add in New Project > Assign team don’t persist. When the project is created, they don’t show up on the Team tab.

**Review by amenity — CHANGES_REQUESTED** [review]
Looks so great! Just need to address the disappearing project team.

**Review by amenity — APPROVED** [review]

**Pull request merged**